extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/index.tsx

```diff
@@ -2,10 +2,12 @@ import React from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { Asset } from "stellar-sdk";
 import get from "lodash/get";
-import { getAssetFromCanonical } from "helpers/stellar";
+import { Icon, TextLink } from "@stellar/design-system";
 
-import { Icon, InfoBlock, TextLink } from "@stellar/design-system";
+import { InfoBlock } from "popup/basics/InfoBlock";
 import { Button } from "popup/basics/buttons/Button";
+
+import { getAssetFromCanonical } from "helpers/stellar";
```

Comment:
@acharb sorry if this is annoying - but I rearranged the imports here. It's good practice to put external libs at the top (we usually enforce this through linting, but can't with a yarn workspace)

Also, grouped some imports together (ex: basics with basics, etc)

It helps find things a little faster when the dep list gets really long

Here's the link to an explanation in our style guide. https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#imports

We're not great about keeping an *exact* order, but we try to at least group similar imports together

Reply:
ya  I've tried being consistent with that but def missed some spots, thanks for fixing!